ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/PointSchedule.hs

```diff
@@ -208,15 +215,20 @@ instance Functor Peers where
 
 -- | A tick is an entry in a 'PointSchedule', containing the peer that is
 -- going to change state.
-newtype Tick =
+data Tick =
   Tick {
-    active :: Peer NodeState
+    active   :: Peer NodeState,
+    duration :: Maybe DiffTime
   }
   deriving (Eq, Show)
```

Comment:
Relatedly, the docs should explain what duration is. Probably the time between the start of this tick and the next (?). This transforms the ticks into time intervals, rather than instantaneous events.

Reply:
FTR the `Maybe` was removed, we use the global tick duration on construction now